tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Test vCenter host connection role
+  block:
+    - name: Add Resource Pool
+      ansible.builtin.include_tasks: manage_resource_pool_role.yml
```

Comment:
I wouldn't call it `manage_resource_pool_role.yml` since it's not a role

Reply:
yes, you are right)